content/providers/01-ai-sdk-providers/02-openai.mdx

```diff
@@ -876,3 +876,68 @@ The following provider options are available:
 | `whisper-1`              | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> |
 | `gpt-4o-mini-transcribe` | <Check size={18} /> | <Cross size={18} /> | <Cross size={18} /> | <Cross size={18} /> |
 | `gpt-4o-transcribe`      | <Check size={18} /> | <Cross size={18} /> | <Cross size={18} /> | <Cross size={18} /> |
+
+## Speech Models
+
+You can create models that call the [OpenAI speech API](https://platform.openai.com/docs/api-reference/audio/speech)
+using the `.speech()` factory method.
+
+The first argument is the model id e.g. `tts-1`.
+
+```ts
+const model = openai.speech('tts-1');
+```
+
+OpenAI speech models support a `text` argument which can be a `string`.
+
+```ts
+const model = openai.speech('tts-1', {
+  text: 'Hello, world!',
+});
```

Comment:
yes this looks like it should be avoided

Reply:
Typo, sorry